client/reader/following-edit/index.jsx

```diff
@@ -326,19 +345,37 @@ var FollowingEdit = React.createClass( {
 			searchPlaceholder = this.translate( 'Search' );
 		}
 
+		const containerClasses = classnames( {
+			'is-adding': this.state.isAddingOpen
+		}, 'following-edit' );
+
 		return (
-			<Main className="following-edit">
+			<Main className={ containerClasses }>
 				<MobileBackToSidebar>
 					<h1>{ this.translate( 'Manage Followed Sites' ) }</h1>
 				</MobileBackToSidebar>
 				{ this.renderFollowError() }
 				{ this.renderUnfollowError() }
+
+				<SectionHeader className="following-edit-header" label={ this.translate( 'Sites' ) } count={ this.state.totalSubscriptions }>
+					<div className="following-edit__more-options">
+						<Gridicon icon="ellipsis" />
+					</div>
+
+					<FollowingEditSortControls onSelectChange={ this.handleSortOrderChange } sortOrder={ this.state.sortOrder } />
+
+					<SectionHeaderButton onClick={ this.toggleAddSite }>
```

Comment:
`SectionHeader` button is deprecated, just using a regular Button inside it with the `compact` prop should be the same.